front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -258,7 +340,10 @@ const IdeaCards = ({ ideaQueryParameters, onUpdateQuery }: Props) => {
             </>
           )}
 
-          <AboveContent filterColumnWidth={filterColumnWidth}>
+          <AboveContent
+            filterColumnWidth={filterColumnWidth}
+            isMapView={selectedView === 'map'}
```

Comment:
We're seeing a lot of "view is map" conditions. Could be worth splitting into a version for each view mode (when we do clean-up).